.circleci/config.yml

```diff
@@ -10,15 +10,9 @@ workflows:
   # See: https://ideas.circleci.com/ideas/CCI-I-627
   on-commit:
     jobs:
-      - update-shortlinks:
-          filters:
-            branches:
-              only: master
-
   # scheduled workflow
   every-15-min:
```

Comment:
Probably this whole block needs to be commented out since there is no `job`.

Reply:
removed